docs/config.md

```diff
@@ -53,6 +53,22 @@ Overrides `CARTESI_AUTH_AWS_KMS_*`.
 
 * **Type:** `string`
 
+## `CARTESI_AUTH_PRIVATE_KEY`
+
+The node will use this private key to sign transactions.
```

Comment:
I suggest adding the format expected:

Ex.: CARTESI_AUTH_PRIVATE_KEY_FILE=59c6995e998f97a5a0044966f0945389dc9e86dae88c7a8412f4603b6b78690d

Reply:
No other option has example. So I didn't know how to add that.